charts/consul/values.yaml

```diff
@@ -205,6 +206,9 @@ global:
       # Please refer to [Vault ACL policies](https://developer.hashicorp.com/consul/docs/connect/ca/vault#vault-acl-policies)
       # documentation for information on how to configure the Vault policies.
       connectCA:
+        # Vault namespace
+        namespace: ""
+
         # The address of the Vault server.
         address: ""
 
```

Comment:
(Outside comment range) Below, I think we'll want to remove the `"namespace"` KV from the `additionalConfig` example so as not to confuse operators about where it should be configured

Reply:
Do you mean remove from additional config?